src/models/index.ts

```diff
@@ -0,0 +1,8 @@
+import { PaUser as PaUserModel } from "./PA_User";
+import { PublicAdministration as PublicAdministrationModel } from "./PublicAdministration";
+import { User as UserModel, UserType as UserTypeEnum } from "./User";
+
+export const PaUser = PaUserModel;
```

Comment:
why we need to create these aliases?

Reply:
Because of the dependencies among the models, `sequelize` must initialize all the models involved in an association before using it, also the ones related to the associative entities. For example, if we want to add a user to an organization, even if the directly involved models are `User` and `Organization`, yet also the model of `OrganizationUser` must be initalized because it takes part in their association, otherwise an error is thrown.
The initialization of a model happens when the model is used, so if we use all the models by importing them in and exporting them from a single file, then we are sure that when we import from that single file all the models are already correctly initialized.